src/lib/ai/models/destroy_addon.ts

```diff
@@ -8,14 +8,14 @@ export default async function (config: Config, addon: Heroku.AddOn, force = fals
   const addonName = addon.name || ''
   const herokuClient = new APIClient(config)
 
-  ux.action.start(`Destroying ${color.addon(addonName)} in the background.\nThe app will restart when complete...`)
+  ux.action.start(`Destroying ${color.addon(addonName)} in the background.\nThe app will restart when complete... `)
 
   await herokuClient.delete<Heroku.AddOn>(`/apps/${addon.app?.id}/addons/${addon.id}`, {
     headers: {'Accept-Expansion': 'plan'},
     body: {force},
   }).catch(error => {
     ux.action.stop('')
-    const error_ = error.body && error.body.message ? new Error(`The add-on was unable to be destroyed: ${error.body.message}.`) : new Error(`The add-on was unable to be destroyed: ${error}.`)
+    const error_ = error.body && error.body.message ? new Error(`The add-on was unable to be destroyed: ${error.body.message}. `) : new Error(`The add-on was unable to be destroyed: ${error}. `)
```

Comment:
```suggestion
    const error_ = error.body && error.body.message ? new Error(`${color.addon(addonName)} was unable to be destroyed: ${error.body.message}. `) : new Error(`${color.addon(addonName)} was unable to be destroyed: ${error}. `)
```

Is this version possible to be more specific?